Tools/TestWebKitAPI/Tests/WebKitCocoa/LocalStorageClear.mm

```diff
@@ -90,118 +90,8 @@ - (void)userContentController:(WKUserContentController *)userContentController d
     TestWebKitAPI::Util::run(&readyToContinue);
 }
 
-static long long fileSize(NSURL* url)
-{
-    NSError *error = nil;
-    NSDictionary *fileAttributes = [[NSFileManager defaultManager] attributesOfItemAtPath:url.path error:&error];
-    if (error)
-        return -1;
-    return [[fileAttributes objectForKey:NSFileSize] longLongValue];
-}
 
 NSString *swizzledBundleIdentifierWebBookmarksD()
 {
     return @"com.apple.webbookmarksd";
 }
-
-NSString *defaultWebsiteCacheDirectory()
-{
-#if PLATFORM(IOS_FAMILY)
-    return nil;
-#else
-    return @"~/Library/Caches/com.apple.WebKit.TestWebKitAPI/WebKit";
-#endif
-}
-
-NSString *defaultApplicationCacheDirectory()
-{
-#if PLATFORM(IOS_FAMILY)
-    // This is to mirror a quirk in WebsiteDataStore::defaultApplicationCacheDirectory and catch any regressions.
-    return [NSHomeDirectory() stringByAppendingPathComponent:@"Library/Caches/com.apple.WebAppCache"];
-#else
-    return [defaultWebsiteCacheDirectory() stringByAppendingString:@"/OfflineWebApplicationCache"];
-#endif
-}
-
-TEST(WKWebView, ClearAppCache)
```

Comment:
This test and database files should probably be kept, as we will need to verify the remaining AppCache data on disk can be removed via API.

Reply:
Right, this test should be removed when we remove the NetworkProcessProxy related code, which should be done in a follow-up, I will revert.

Reply:
As of AppCache data, given the time it has been obsolete, I would hope we do not need to keep the NetworkProcessProxy related code around